docs/src/main/tut/pull_request.md

```diff
@@ -128,6 +130,57 @@ createPullRequestIssue.exec[cats.Id, HttpResponse[String]]() match {
 
 See [the API doc](https://developer.github.com/v3/pulls/#create-a-pull-request) for full reference.
 
+## List pull requests
+
+You can list the reviews for a pull request using `listReviews`, it takes as arguments:
+
+- the repository coordinates (`owner` and `name` of the repository).
+- the pull request id.
+
+As an example, if we wanted to see all the reviews for pull request 139 of `47deg/github4s`:
+
+```scala
+val listReviews = Github(accessToken).pullRequests.listReviews(
+  "47deg",
+  "github4s",
+  139)
+
+listReviews.exec[cats.Id, HttpResponse[String]]() match {
+  case Left(e) => println("Something went wrong: s{e.getMessage}")
+  case Right(r) => println(r.result)
+}
+```
+
+The `result` on the right is the matching [List[PullRequestReview]][pr-scala].
+
+## Get an individual review
+
+You can list the reviews for a pull request using `listReviews`, it takes as arguments:
+
+- the repository coordinates (`owner` and `name` of the repository).
+- the pull request id.
+- the review id.
+
+As an example, if we wanted to see review 39355613 for pull request 139 of `47deg/github4s`:
+
+```scala
+val review = Github(accessToken).pullRequests.getReview(
+  "47deg",
+  "github4s",
+  139,
+  39355613)
+
+review.exec[cats.Id, HttpResponse[String]]() match {
+  case Left(e) => println("Something went wrong: s{e.getMessage}")
+  case Right(r) => println(r.result)
+}
+```
+
+The `result` on the right is the matching [PullRequestReview][pr-scala].
+
+
+See [the API doc](https://developer.github.com/v3/pulls/reviews/#list-reviews-on-a-pull-request) for full reference.
```

Comment:
We usually have a link to the gh doc per endpoint, so this line should be in the `# List reviews` section and here the link should be changed to https://developer.github.com/v3/pulls/reviews/#get-a-single-review